LenovoLegionToolkit.Lib.Automation/CmdLine/IPCMessagePack.cs

```diff
@@ -0,0 +1,14 @@
+﻿using ProtoBuf;
```

Comment:
I would move or CLI related code to WPF project. It is not ideal architecture, but with the assumption that LLT needs to run in background, putting it there makes more sense if this functionality will be extended at some point.

Reply:
Do u mean I should move all IPC stuff to WPF project?